app/operations/shopify/product_operations.rb

```diff
@@ -0,0 +1,27 @@
+module Shopify
+  class ProductOperations
```

Comment:
ProductCRUD ;)

For me it triggers something in my head because I use this term for that behavior on most of my projects. Else, we could maybe name it ProductFactory? I am unsure of a better name TBH.